source/Calamari.Shared/Integration/Certificates/PrivateKeyAccessRuleExtensionMethods.cs

```diff
@@ -1,6 +1,4 @@
-﻿#if WINDOWS_CERTIFICATE_STORE_SUPPORT
```

Comment:
A bunch of the file changes are just this, removing the `#if WINDOWS_CERTIFICATE_STORE_SUPPORT` compiler directive, meaning it compiles into netcore, even if it shouldnt actually be getting invoked by anything (since the `Calamari.netfx` flavour is used for anything on Windows that would have the IIS/WindowsCert variables provided)